src/askscript/__tests__/00-documentation-examples/documentation01-complete_example.test.test.result.ts

```diff
@@ -0,0 +1 @@
+export = null;
```

Comment:
AskScript tests (.test.ask) should be treated more like TypeScript tests (.test.ts) - we shouldn't rely on .test.result.ts and test.args.ts files for them

Reply:
Let's keep the PRs small.  Filing your note as https://github.com/xFAANG/askql/issues/289

Reply:
@czerwinskilukasz1 this I think it makes better sense to fix here, this PR is an enhancement, not a bug fix - so let's make sure that we are stepping in the right direction from the first step